master/pkg/actor/api/ws.go

```diff
@@ -47,7 +47,7 @@ func (w WebSocketConnected) Accept(
 		ctx.Respond(errors.Wrap(err, "websocket connection error"))
 		return nil, false
 	}
-	a, _ := ctx.ActorOf(uuid.New(), WrapSocket(conn, msgType, usePing))
+	a, _ := ctx.ActorOf("websocket", WrapSocket(conn, msgType, usePing))
```

Comment:
blocking: this is going to cause collisions as every connecting agent needs to have a dedicated socket actor. Also its used for events.